docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,404 @@
+                              --save-model-to outputs/openvino/
+
+  ...
+
+  2022-12-29 01:39:11,980 | INFO : Exporting completed
+  2022-12-29 01:39:11,980 | INFO : run task done.
+  2022-12-29 01:39:11,990 | INFO : Exporting completed
+
+
+#TODO show how to run evaluation of exported model?
+
+*************
+Optimization
+*************
+
+1. ``otx optimize`` optimizes a model using NNCF or POT depending on the model format.
+
+- NNCF optimization used for trained snapshots in a framework-specific format such as checkpoint (pth) file from Pytorch. It starts training based on the weights from previous step in fewer precision.
+- POT optimization used for models exported in the OpenVINO IR format. It decreases the precision of the exported model and performs the post-training optimization.
+
+The function results with a following files, which could be used to run ``otx demo``[link]:
+
+- confidence_threshold
+- config.json
+- label_schema.json
+- openvino.bin
+- openvino.xml
+
+Read more about optimization in [#TODO link]
+
+2. Command example for optimizing a PyTorch model (.pth) with OpenVINO NNCF.
+
+.. code-block::
+
+  (detection) ...$ otx optimize otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml 
+                                --train-ann-files BBCD/train/_annotations.coco.json 
+                                --train-data-roots  BBCD/train 
+                                --val-ann-files BBCD/valid/_annotations.coco.json 
+                                --val-data-roots BBCD/valid 
+                                --load-weights outputs/weights.pth
+                                --save-model-to outputs/nncf
+                                --save-performance outputs/nncf/performance.json
+
+  ...
+
+  2022-12-29 02:11:49,018 | INFO : Loaded model weights from Task Environment
+  2022-12-29 02:11:49,018 | INFO : Model architecture: ATSS
+  2022-12-29 02:11:49,018 | INFO : Loaded model weights from Task Environment
+  2022-12-29 02:11:49,018 | INFO : Model architecture: ATSS
+  2022-12-29 02:11:49,019 | INFO : Task initialization completed
+  INFO:nncf:Please, provide execution parameters for optimal model initialization
+  2022-12-29 02:11:56,996 - mmdet - INFO - Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  [ INFO ] Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  [ INFO ]  Received non-NNCF checkpoint to start training -- initialization of NNCF fields will be done
+  2022-12-29 02:11:56,999 - mmdet - INFO - Calculating an original model accuracy
+  ...
+
+  INFO:nncf:Original model accuracy: 0.4319
+  INFO:nncf:Compressed model accuracy: 0.5564
+  INFO:nncf:Absolute accuracy drop: -0.1245
+  INFO:nncf:Relative accuracy drop: -28.82%
+  INFO:nncf:Accuracy budget: 0.1345
+
+
+#TODO significant drop of the loaded snapshot: 0.43 instead of 0.83
+
+#TODO show how to run evaluation of optimized model and its metrics?
```

Comment:
After export we see the output metric. I think it is enough, but we can mention, that optimized models can be easily validated just like we did in the  previous sections